### DISCUSSION TITLE
Birds pictures aren't the right ones

### DISCUSSION THREAD

**Discussion by blogdemoi** [mcguirepr89/BirdNET-Pi#950] [Q&A] [answered]
Hi,

I noticed that since two days, the pictures of birds aren't the right ones:

![XKAFihY](https://github.com/mcguirepr89/BirdNET-Pi/assets/83965802/6f83e58a-cd00-4b80-9d6d-864086c40182)

**Comment by ehpersonal38** [comment, marked as answer]
Hi, please update your system, I pushed a fix for this yesterday

**Reply by blogdemoi** [reply]
Thank you for the fix!
